components/modal/demo/position.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 7
-title: 自定义位置
+title: 
+  zh-CN: 自定义位置
+  en-US: To customize position of modal
 ---
 
+## zh-CN
+
 `1.0` 之后，Modal 的 `align` 属性被移除，您可以直接使用 `style.top` 或配合其他样式来设置对话框位置。
 
+## en-US
+
+After `1.0`,  we remove `align` of Modal, you can use `style.top` or other styles to set position of modal dialog.
```

Comment:
After release `1.0`,  Modal's `align` prop was removed. You can use `style.top` or other styles to set position of modal dialog.